tests/unit/test_providers.py

```diff
@@ -10,6 +10,7 @@
 from circuit_maintenance_parser.provider import (
     GenericProvider,
     Cogent,
+    Colt,
```

Comment:
I know this was the pattern before, but this is gonna change, so better if you do not update this file, less conflicts to fix :)

Reply:
Ah ok, I see the other PR now. Will remove.